lib/Comment/styles.scss

```diff
@@ -0,0 +1,56 @@
+.conversation__text__cutoff {
+  &:after {
+    @apply absolute h-8 w-full;
+    content: ' ';
+    left: 40px;
+    top: 96px;
+    background: linear-gradient(
+      to bottom,
+      rgba(255, 255, 255, 0) 0%,
+      rgba(253, 253, 253, 0.5) 50%,
+      #fff 100%
+    );
+  }
+}
+
+.conversation__inactive {
+  &:hover {
+    .conversation__text__cutoff {
+      &:after {
+        background: linear-gradient(
+          to bottom,
+          rgba(255, 255, 255, 0) 0%,
+          rgba(253, 253, 253, 0.5) 50%,
+          #f0f2f5 100%
+        );
+      }
+    }
+  }
+}
+
+.new-comment__input {
+  @apply border-0 p-2;
+}
+
+.edit-comment__input {
+  @apply border-0 pr-2 py-0 pl-0;
+}
+
+.conversation-comment__line {
+  &:before {
+    @apply absolute h-full border-l-2 border-solid border-neutral-90 border-r-0 border-b-0;
+    content: ' ';
+    top: 38px;
+    left: 22px;
+  }
+}
+
+.conversation-blur-content {
+  background: rgba(255, 255, 255, 0.75);
+
+  @supports (filter: blur()) {
+    transition: filter $animation-time-micro/2;
+    filter: blur(4px);
+    background: none;
+  }
+}
```

Comment:
This is the blur for when you click `Delete` on a comment, I can't seem to find a solution for most of these styles in tailwind.

Reply:
Same comments as before, if tailwind is our new master, the designs should heed to its technical limitations.

Reply:
Again missing styles could probably be added. For something like this maybe doing something like this would be neat https://tailwindcss.com/docs/extracting-components/#extracting-css-components-with-apply to have a `.blur` class we can use :)